pkg/ingester/limiter.go

```diff
@@ -105,24 +107,33 @@ func (l *Limiter) minNonZero(first, second int) int {
 	return first
 }
 
-func (l *Limiter) convertGlobalToLocalLimit(globalLimit int) int {
+func (l *Limiter) convertGlobalToLocalLimit(globalLimit, maxStreamsPerUser int) int {
```

Comment:
we do not need to pass local limit here, local limit is used on line number 92 .... 
local limit is a limit per ingester, regardless how many ingesters we have in the ring.